src/main/java/Duke.java

```diff
@@ -1,10 +1,135 @@
+import java.util.ArrayList;
+import java.util.Scanner;
+
+/**
+ * Driver class that takes in input and performs certain functions based on input.
+ */
 public class Duke {
+
+    protected static ArrayList<Task> taskList = new ArrayList<>();
+
+    /**
+     * Main class to take in user input.
+     * @param args Filler
+     */
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        Scanner sc = new Scanner(System.in);
+
+        String command = "Hello";
+
+        while (!command.equalsIgnoreCase("bye")) {
+            if (command.equals("Hello")) {
+                System.out.println("Hello! I'm Duke");
+                System.out.println("What can I do for you?");
+            } else if (command.equals("list")) {
+                enumerateTasks();
+            } else if (command.startsWith("done")) {
+                String[] delString = command.split("\\s+");
```

Comment:
delString is not a very intuitive variable name, maybe you could make it more specific?